source/wp-content/themes/wporg-showcase-2022/src/archive-results-context/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import ServerSideRender from '@wordpress/server-side-render';
+import { useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+
+function Edit( { attributes, name } ) {
+	return (
+		<div { ...useBlockProps() }>
+			<ServerSideRender block={ name } attributes={ attributes } />
+		</div>
```

Comment:
It seems like this wrapper in JS is not needed, since the rendered block also contains the block props attributes.

I added this to a page to test out the attributes, and noticed my "3X-Small" padding is there twice.
<img width="329" alt="Screenshot 2022-11-07 at 3 39 53 PM" src="https://user-images.githubusercontent.com/541093/200430626-b1df1b12-6660-4d23-bf55-4e464dfe84e0.png">

```suggestion
		<ServerSideRender block={ name } attributes={ attributes } />
```